lib/net.c

```diff
@@ -1878,6 +1915,87 @@ check_if_rss_key_update(const struct gatekeeper_if *iface)
 	return 0;
 }
 
+static uint32_t
+count_macs(const char *if_name, const struct rte_ether_addr *macaddrs,
+	uint32_t max_mac_addrs)
+{
+	bool ended = false;
+	uint32_t i, count = 0;
+
+	for (i = 0; i < max_mac_addrs; i++) {
+		if (rte_is_zero_ether_addr(&macaddrs[i])) {
+			ended = true;
+			continue;
+		}
+		if (unlikely(ended)) {
+			G_LOG(ERR, "%s(%s): MAC address "
```

Comment:
Replace `MAC address` with `MAC` to match the log entry ` "%s(%s): MAC [%u/%u] " RTE_ETHER_ADDR_PRT_FMT "\n"`.